marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/ingest/default/main.sjs

```diff
@@ -27,13 +23,25 @@ function main(id, content, options) {
     }
   }
 
-  let headers = {};
-
+  let headers = createHeaders(id, content, options);
   let envelope = datahub.flow.flowUtils.makeEnvelope(instance, headers, triples, outputFormat);
 
   return envelope;
 }
 
+function createHeaders(id, content, options) {
+  let headers = {};
+  for (let key in options.headers) {
+    headers[key] = options.headers[key];
+    if (headers[key] == datahub.flow.consts.CURRENT_DATE_TIME) {
+      headers[key] = fn.currentDateTime;
```

Comment:
should be fn.currentDateTime()

Reply:
yes. i missed it when I copied from qconsole.